sdk/src/types/tradeHistory.ts

```diff
@@ -92,6 +94,8 @@ export type PositionTradeAction = {
   reasonBytes?: string | Uint8Array;
   shouldUnwrapNativeToken: boolean;
   liquidationFeeAmount?: bigint;
+  twapGroupId: string | undefined;
+  numberOfParts: number | undefined;
```

Comment:
probably it's better to create an optional object for twap params with required fields, otherwise it can require an extra checks